pkg/proxy/proxy.go

```diff
@@ -111,11 +111,24 @@ func (p *Proxy) RoundTrip(req *http.Request) (*http.Response, error) {
 	info, ok, err := p.reqAuther.AuthenticateRequest(req)
 	if err != nil {
 		klog.Errorf("unable to authenticate the request due to an error: %v", err)
+
+		if !ok {
+			klog.Info("not a valid OIDC token, forward to Kubernetes API as this may be a valid API token")
+			return p.clientTransport.RoundTrip(req)
+		}
+
```

Comment:
I'm a little confused about this case, is it possible for `AuthenticateRequest` to return an error and `ok = true`?

Reply:
I'm not aware of a scenario where `err != nil & ok`